app/team.go

```diff
@@ -1165,7 +1165,7 @@ func (a *App) RestoreTeam(teamId string) *model.AppError {
 
 func (a *App) GetTeamStats(teamId string) (*model.TeamStats, *model.AppError) {
 	tchan := a.Srv.Store.Team().GetTotalMemberCount(teamId)
-	achan := a.Srv.Store.Team().GetActiveMemberCount(teamId)
```

Comment:
We were doing it concurrently (actually it keeps been concurrent execution) but I think i prefer to use the explicit async call here (with the channel and the gorouting, see the example PR).